app/routes/achievements/components/Overview.module.css

```diff
@@ -0,0 +1,26 @@
+@import url('~app/styles/variables.css');
+
+.listWrapper {
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--spacing-md);
+
+  @media (--mobile-device) {
```

Comment:
`--mobile-device` is a bit large (`992px`) - probably want to use something like `--small-viewport`